src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -419,7 +408,8 @@
                      :transform [{:translate-y content-translate-y}]}
                     {})}
            (case @active-tab
-             1 [scan-qr-code-tab qr-view-finder request-camera-permission]
+             1 [scan-qr-code-tab qr-view-finder
```

Comment:
@OmarBasem - Kindly remove the changes on lines 411 and 412. 
The `request-camera-permission` has to be removed as it's already a separate function. I have cleaned up in a PR https://github.com/status-im/status-mobile/pull/16558/files#r1259797040